packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -273,34 +273,18 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
       !isOpen && isAnimating && isOnRightSide && AnimationClassNames.slideRightOut40,
       focusTrapZoneClassName
     ],
-    commands: [
-      classNames.commands,
-      {
-        marginTop: 18
-      },
-      hasCustomNavigation && {
-        marginTop: 'inherit'
-      }
-    ],
+    commands: [classNames.commands],
     navigation: [
       classNames.navigation,
       {
         display: 'flex',
-        justifyContent: 'flex-end'
+        justifyContent: 'space-between'
       },
       hasCustomNavigation && {
         height: commandBarHeight
```

Comment:
I would double check this, but changing justifyContent: 'space-between' might be breaking customNavigation which actually needs flex-end

Reply:
What does custom nav look like? flex-end is going to cause the title to right-align. You can try putting flex-grow:1 on the title to make sure it stays left aligned, since then it'll fill up all that left-hand side space and the text inside will left-align to that. Just have to make sure the flex parent there is always 100% width, no more and no less.

Reply:
Why does customNavigation need flex-end?

Reply:
@aneeshack4 @phkuo it doesn't! just checked